### PR TITLE
CTDA-1291 Sort by lastUpdated and filter number of departures returned

### PR DESCRIPTION
**Pull request by crystalmark**

### CODE REVIEW

**Review by DavidGregory084 — APPROVED** [review]
👍

File: app/repositories/DepartureRepository.scala [hunk 1]
```diff
@@ -251,10 +251,12 @@ class DepartureRepository @Inject()(mongo: ReactiveMongoApi, appConfig: AppConfi
   }
 
   def fetchAllDepartures(eoriNumber: String, channelFilter: ChannelType): Future[Seq[DepartureWithoutMessages]] =
+//    import reactivemongo.play.json.JSONSerializationPack
```
Comment (RobertLaverick) [inline app/repositories/DepartureRepository.scala:254]: please remove

File: app/models/DepartureWithoutMessages.scala [hunk 2]
```diff
@@ -59,6 +59,18 @@ object DepartureWithoutMessages {
...
+      (__ \ "_id").write[DepartureId] and
+        (__ \ "channel").write[ChannelType] and
+        (__ \ "eoriNumber").write[String] and
+        (__ \ "movementReferenceNumber").write[Option[MovementReferenceNumber]] and
+        (__ \ "referenceNumber").write[String] and
+        (__ \ "status").write[DepartureStatus] and
+        (__ \ "created").write(MongoDateTimeFormats.localDateTimeWrite) and
+        (__ \ "updated").write(MongoDateTimeFormats.localDateTimeWrite)
+    )(unlift(DepartureWithoutMessages.unapply))
+
```
Comment (RobertLaverick) [inline app/models/DepartureWithoutMessages.scala:73]: Can you explain why this has been added?

**Review by RobertLaverick — CHANGES_REQUESTED** [review]

File: app/repositories/DepartureRepository.scala (hunk 1 above)
Comment (crystalmark) [reply app/repositories/DepartureRepository.scala]: Done!

File: app/models/DepartureWithoutMessages.scala (hunk 2 above)
Comment (crystalmark) [reply app/models/DepartureWithoutMessages.scala]: Twas a test - removed now

**Review by RobertLaverick — APPROVED** [review]

**Review by amaalali — APPROVED** [review]

**Review by RobertLaverick — APPROVED** [review]

**Pull request merged**